Simulation/Classes/StateVector_Index.m

```diff
@@ -0,0 +1,24 @@
+%% State Vector Indecies

```

Comment:
I think we've had this before - the red `^M` character, seems to be mixing of CRLF Windows line ending and LF Unix ones, [per this issue](https://github.com/atom/atom/issues/3105).

`Indecies` is also misspelled a few times in this file.